Models/reviews.models.js

```diff
@@ -11,5 +11,14 @@ exports.fetchReview = (review_id) => {
        return review;
       }
 
+    })
+}
+
+exports.updateReview = (newVotes, review_id) => {
+    return db.query("UPDATE reviews SET votes = $1 WHERE review_id = $2 RETURNING *", [newVotes, review_id] )
+    .then((result) => {
```

Comment:
Have a look at whether we can set the votes with some sort of additional operation